calculator.go

```diff
@@ -1,13 +1,101 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"fmt"
+	"go/token"
+	"go/types"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+// Add takes two or more numbers and returns the result of adding them together.
+func Add(a, b float64, extra ...float64) float64 {
+	total := a + b
+	for _, n := range extra {
+		total += n
+	}
+	return total
 }
 
-// Subtract takes two numbers and returns the result of subtracting the second
+// Subtract takes two or more numbers and returns the result of subtracting the second and subsequent numbers from the first.
 // from the first.
-func Subtract(a, b float64) float64 {
-	return b - a
+func Subtract(a, b float64, extra ...float64) float64 {
+	result := a - b
+	for _, n := range extra {
+		result -= n
+	}
+	return result
+}
+
+// Multiply takes two or more numbers and returns the result of multiplying them together.
+func Multiply(a, b float64, extra ...float64) float64 {
+	result := a * b
+	for _, n := range extra {
+		result *= n
+	}
+	return result
+}
+
+// Divide takes two or more numbers and returns the result of dividing the first by the second and subsequent numbers, or an error if division by zero occurs.
```

Comment:
It's a good idea to wrap your doc comments at about 80 characters; there's an excellent VS Code extension for this called [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap). You just write your comment, hit Alt-Q, and it will be automatically reformatted across as many lines as necessary!